roles/openshift_facts/vars/Fedora.yml

```diff
@@ -0,0 +1,7 @@
+---
+required_packages:
+  - iproute
+  - python3-dbus
+  - python-six
```

Comment:
I suspect that here we want to have `python3-six` package but it works for me without this. Perhaps, because I used non-minimal Fedora template.